README.md

```diff
@@ -72,13 +72,19 @@ _Note:_ Deployment strategy options currently only apply to Webservice/Websocket
 ```yaml
 options:
   merge: true,
-  safeNamespace: vendor
+  safeNamespace: vendor,
+  safeConfigs:
```

Comment:
how to define multiple parameters in a service?
```
safeConfigs:
- webservice.name
- webservice.title
```
like this?

Reply:
yes